replication-api-impl/src/test/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImplTest.java

```diff
@@ -65,7 +65,7 @@ public void writeToMap() {
     assertThat(props.get(FILTER_KEY), is("filter"));
```

Comment:
❓ Should we add a test for reading from version 1 to version 2?

Reply:
It was kind of already doing that but I added a more explicit test for it